TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -6266,7 +6266,7 @@ setSendingField:
 // Format 2: SET for adjusting indexes
 
 setStatementForIndexes:
-	SET indexStorageArea+ (UP | DOWN) BY integerVariable1;
+	SET indexStorageArea+ (UP | DOWN) BY (variableOrExpression2);
```

Comment:
This change has to be made into TypeCobolCodeElement grammar

Reply:
setStatementForIndexes:
	SET indexStorageArea+ (UP | DOWN) BY integerVariable1;

Have to be in CobolCodeElements.g4 and

 setStatementForIndexes:
	SET indexStorageArea+ (UP | DOWN) BY variableOrExpression2;

will overide it in TypeCobolCodeElement.g4 for keeping a correct Cobol85/2002 parsing.
Is that the goal ?

Reply:
Yes that's the goal